SeedService/src/SeedService_service.cc

```diff
@@ -317,6 +320,12 @@ namespace mu2e {
     state_.clear();
   }
 
+  void SeedService::postEvent             (art::Event const& ev, art::ScheduleContext context) {
+    if(verbosity_ > 1) {
+      std::cout << "SeedService postEvent Event " << ev.id() << " current seed " << currentSeed_ << std::endl;
```

Comment:
Do you want to leave this in after you showed it is not meaningful?

Reply:
my bad, accidentally added this file.  removing it now.